launch/launch/frontend/parser.py

```diff
@@ -183,3 +195,8 @@ def load(
         finally:
             if didopen:
                 fileobj.close()
+
+    @classmethod
+    def get_file_extensions(cls) -> Set[Text]:
+        """Return a set of file extensions for this parser."""
```

Comment:
@christophebedard nit: 
```suggestion
        """Return the set of file extensions known to this parser."""
```

Reply:
changed in bb5538ffc4ed9cc5bfbf9d78a999d3a29c79a688